src/style.css

```diff
@@ -1,3 +1,78 @@
+:root {
+  --blue: #00acdc;
+  --black: #000;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+}
+
+.header__logo {
+  height: 60px;
+}
+
+.header__img {
+  margin: 10px 0;
+}
+
+.nav {
+  height: 60px;
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  list-style: none;
+  padding-left: 0;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  display: block;
+  color: var(--black);
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+}
+
+.nav__item:nth-child(4) .nav__link:hover {
+  color: var(--blue);
+  font-weight: 500;
+}
+
+.is-active {
+  position: relative;
+  color: var(--blue);
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4px;
+  border-radius: 2px;
```

Comment:
8px, isn't it?

Reply:
The same as above

Reply:
check figma design. there is should be `8px`

Reply:
<img width="1199" alt="Screenshot 2024-01-08 at 18 54 29" src="https://github.com/mate-academy/layout_moyo-header/assets/117587463/a926bff1-241f-4d45-bf57-eb9fd9a6d5e6">